webdriver/tests/get_active_element/META.yml

```diff
@@ -0,0 +1,5 @@
+links:
+    - product: chrome
+      url: https://crbug.com/1020018
```

Comment:
This is about the test being flaky, but yeah, let's link to it, could be relevant.